apps/web/src/app/styles/fonts.css

```
@@ -3,8 +3,17 @@
  */
 @import url('https://cdn.jsdelivr.net/gh/orioncactus/pretendard@v1.3.9/dist/web/variable/pretendardvariable-dynamic-subset.min.css');
 
+@font-face {
+  font-family: 'Waguri';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Waguri" (font-family-name-quotes)

(font-family-name-quotes)